### DISCUSSION TITLE
error after registering action

### DISCUSSION THREAD

**Discussion by ritvic0711** [NVIDIA-NeMo/Guardrails#515] [Q&A]
I face the below error for 
response = await rails.generate_async(
    messages=[{
        "role": "user",
        "content": "Hey, my name is Zoltan and my phone number is +49 30 065615973."
    }])
    
WARNING:nemoguardrails.actions.action_dispatcher:Error while execution generate_bot_message: expected token 'end of print statement', got '='
{'role': 'assistant', 'content': "I'm sorry, an internal error has occurred."}

**Comment by drazvan** [comment]
Hi @ritvic0711! Can you create an issue for this and provide more details regarding your configuration (e.g., what LLM, the full log in verbose mode). Thanks!
